extensions/amp-story/0.1/system-layer.js

```diff
@@ -220,6 +227,14 @@ export class SystemLayer {
    * @param {!Event} e
    * @private
    */
+  onEscapeKey(e) {
+    this.dispatch_(EventType.CLOSE_BOOKEND, e);
```

Comment:
Can you add that this PR fixes #76?

Reply:
Sure thanks, I didn't see the issue. I just got tired of moving my mouse to the X button : P

Reply:
Done.